plugins/bc-uploader/tests/TestCase/Controller/Admin/UploaderCategoriesControllerTest.php

```diff
@@ -143,4 +145,35 @@ public function testAfterEditEvent()
         $query = $uploaderCategories->find()->where(['name' => 'afterAdd']);
         $this->assertEquals(1, $query->count());
     }
+
+    /**
+     * test edit
+     *
+     */
+    public function test_edit()
+    {
+        $this->loadFixtureScenario(UploaderCategoriesScenario::class);
+
+        $this->enableSecurityToken();
+        $this->enableCsrfToken();
+
+        //正常系実行
+        $data = [
+            'name' => 'japan'
+        ];
+        $this->post('/baser/admin/bc-uploader/uploader_categories/edit/1', $data);
```

Comment:
``$this->post('/baser/admin/bc-uploader/uploader_categories/edit/1', ['name' => 'japan']);`
`@thangnnmd vì tham số là 1 mảng có 1 thằng thôi, nên a viết thế này cho ngắn nè,

Reply:
anh sửa rồi nhé